Firebase/InAppMessagingDisplay/Public/FirebaseInAppMessagingDisplay.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRIAMDefaultDisplayImpl.h"
```

Comment:
We don't have anything to expose as public interface in this Display SDK itself. So we can skip having this line.

Reply:
Shouldn't it be included if it's in the Public directory though? If not, why is the interface in a Public directory?

Reply:
Actually, checking the comments https://github.com/firebase/firebase-ios-sdk/blob/master/Firebase/InAppMessagingDisplay/Public/FIRIAMDefaultDisplayImpl.h#L23 again, it's used by our testing app for UI tests. Let's keep it then. 

Sorry about the confusion

Reply:
SGTM, thanks.